packages/curve-ui-kit/src/shared/icons/ChainIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { CURVE_ASSETS_URL } from 'ui'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { styled } from '@mui/material/styles'
+import Image from 'next/image'
+
+const { IconSize } = SizesAndSpaces
+
+const Icon = styled(Image)({})
+
+export const ChainIcon = ({ blockchainId, size }: { blockchainId: string; size: keyof typeof IconSize }) => {
```

Comment:
Not super related to this PR, but do you think this component can replace `curve-ui-kit\src\features\switch-chain\ui\ChainIcon.tsx`?

Reply:
Yeah I've been looking into it. In this case we have less data so it's tricky. I'll try

Reply:
yeah the one in the header needs to have a fixed height too, not responsive